fairroot/alignment/FairAlignmentHandler.cxx

```diff
@@ -100,15 +101,15 @@ void FairAlignmentHandler::AddAlignmentMatrices(const std::map<std::string, TGeo
 
 void FairAlignmentHandler::RecomputePhysicalAssmbBbox() const
 {
```

Comment:
Could you put some sort of comment here, so that we know in the future, why this all is happening?
* Consider linking to the ROOT issue, so that we can check in the future, if that issue has been resolved and in which version of ROOT.

Reply:
I added both some descriptive information in the method itself and a reference to the ROOT issue in the existing `FIXME` to make it clear that this may be included on the ROOT side in the `Refresh` which we are anyway calling two line later.
Just let me know in case I should also add a line on why the Assemblies need a recompute at all.

Reply:
I asked about this, as we will likely keep these fixes for somethign like three years or more (assuming ROOT fixes it just now), because we will support "older" ROOT releases for a while.

@karabowi @fuhlig1: Are the current comments good enough for you?